src/main/java/org/n52/iceland/ogc/ows/ServiceMetadataRepository.java

```diff
@@ -22,8 +22,6 @@
 import org.n52.iceland.util.LocalizedProducer;
 
 /**
- * TODO JavaDoc
- *
```

Comment:
It's your ``TODO``, don't complain!

The repositories would profit more from a well written REAMDE imho...